evergreen/run-tests.sh

```diff
@@ -70,8 +80,12 @@ fi
 
 echo "Running $AUTH tests over $SSL for $TOPOLOGY with $COMPRESSOR compressor and connecting to $MONGODB_URI"
 
-if [ "$OS" == "windows-64" ]; then
+if [[ "$OS" =~ Windows|windows ]]; then
   export TARGET="Test"
+  if [ "$OCSP_TLS_SHOULD_SUCCEED" != "nil" ]; then
+    export TARGET="TestOcsp"
```

Comment:
who is the consumer of this variable?

Reply:
I think I got it. It determines which task should be called in build.cake

Reply:
Correct! :)